loop-list.php

```diff
@@ -1,65 +1,47 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=yes, shrink-to-fit=no" />
-    <title>Blog Lists</title>
-
-    <!-- <link rel="stylesheet" media="all" href="https://cdn0.vox-cdn.com/stylesheets/vox.v65a8a7248c298ad4.css" /> -->
-    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
-    <link href="https://fonts.googleapis.com/css?family=Fira+Sans" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab:400,700" rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" type="text/css" href="<?php echo get_template_directory_uri(); ?>/css/list-style.css">
-
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
-    <script type='text/javascript' src='<?php echo get_template_directory_uri(); ?>/js/vectordiv.js'></script>
-    <script src="<?php echo get_template_directory_uri(); ?>/js/list-script.js"></script>
-
-  </head>
-  <body>
-
-      <div id="bg">
-        <img src="http://media.gettyimages.com/videos/pink-white-and-light-gray-clouds-pass-through-a-baby-blue-sky-video-id72683500?s=640x640" alt="">
-      </div>
-
-
-    <div class="container-fluid">
-    <div class="row">
-
-      <div class="col-md-3 panel panel-default sidebar1">
-        <div class="panel-body">
-          <h2><center>Share</center></h2>
-        </div>
-
+<?php get_header(); ?>
+
+  <script type='text/javascript' src='<?php echo get_template_directory_uri(); ?>/js/vectordiv.js'></script>
+  <script src="<?php echo get_template_directory_uri(); ?>/js/scrollToFixed.js"></script>
+  <script src="<?php echo get_template_directory_uri(); ?>/js/list-script.js"></script>
```

Comment:
Hmm... Since scrollToFixed.js is just the plugin code, I'd actually like to keep that as a separate file from our custom code. This is something we should discuss more though as we start to organize the js folder, but I'd say leave it for now.